manim/config/config_utils.py

```diff
@@ -349,25 +349,27 @@ def _parse_cli(arg_list, input=True):
     # The following are mutually exclusive and each overrides
     # FRAME_RATE, PIXEL_HEIGHT, and PIXEL_WIDTH,
     parser.add_argument(
-        "-l",
```

Comment:
I'm not sure we should completely remove these flags just yet. Doing so will most likely break every single user's workflow. Yes, it's an easy change, but it still needs to be done with care.

Is there a way of adding a deprecation warning instead?

Reply:
I don't think it is possible to tell `-l` apart from `--low_quality`. If we want to warn for deprecation we could maybe create a new argument with just `-l` and if someone uses it warn them. Not sure if that will produce clean code though.

Reply:
Hmm that's a good point. I'm still a bit weary of breaking every user's flow though. Let's see what others think.